configure

```diff
@@ -1123,6 +1128,9 @@ for ac_option in $@; do
 	--disable-faad)               _faad=no               ;;
 	--enable-faad)                _faad=yes              ;;
 	--disable-fluidsynth)         _fluidsynth=no         ;;
+	--enable-fluidsynth)          _fluidsynth=yes        ;;
+	--disable-fluidlite)          _fluidlite=no          ;;
+	--enable-fluidsynth)          _fluidlite=yes         ;;
 	--enable-readline)            _readline=yes          ;;
```

Comment:
Shouldn't this read `--enable-fluidlite` instead?

Reply:
Fixed.